addons/im_livechat/static/src/models/public_livechat_global.js

```diff
@@ -83,6 +83,15 @@ registerModel({
             }
             return clear();
         },
+        _computeWelcomeMessages() {
+            const welcomeMessages = this.messaging.publicLivechatGlobal.messages.filter((message) => {
```

Comment:
this.messaging.publicLivechatGlobal = this

Reply:
change done